src/lib/Client.ts

```diff
@@ -0,0 +1,436 @@
+    const { host, headers } = this;
+    const url = `${host}/address-verification`;
+    return axios.post(url, bodyParams, { headers });
+  };
+
+  /**
+   * @param bodyParams 
+   * @returns information about the verified routing number
+   * 
+   * {@link [Verify Routing Number](https://docs.synapsefi.com/api-references/miscellaneous/verify-routing-number)}
+   */
+  verifyRoutingNumber(
+    bodyParams: { routing_num?: string; type?: string } = {},
+  ): Promise<AxiosResponse<IVerifiedRoutingNum>> {
+    const { host, headers } = this;
+    const url = `${host}/routing-number-verification`;
+    return axios.post(url, bodyParams, { headers });
+  };
+
+  // GET CRYPTO QUOTES
+  getCryptoQuotes() {
+    const { host, headers } = this;
+    const url = `${host}/nodes/crypto-quotes`;
+
+    return axios.get(url, { headers });
+  };
+
+  // GET CRYPTO MARKET DATA
+  getCryptoMarketData(queryParams: IQueryParams = {}) {
+    const { limit, currency } = queryParams;
+    const { host, headers } = this;
+    const originalUrl = `${host}/nodes/crypto-market-watch`;
+
+    const urlWithParams = addQueryParams({ originalUrl, limit, currency });
+    return axios.get(urlWithParams, { headers });
+  };
+
+  // GET WEBHOOK LOGS
+  getWebhookLogs() {
+    const { host, headers } = this;
+    const url = `${host}/subscriptions/logs`;
+
+    return axios.get(url, { headers });
+  };
+
+  // GET TRADE MARKET DATA
+  getTradeMarketData(queryParams: IQueryParams = {}) {
+    const { ticker } = queryParams;
+    const { host, headers } = this;
+    const originalUrl = `${host}/nodes/trade-market-watch`;
+
+    const url = addQueryParams({ originalUrl, ticker });
+    return axios.get(url, { headers });
+  };
+
+  /**
+   * @description Only INTERCHANGE-US transactions that have been RETURNED within the last 14 days with return code of IR999 can be disputed.
+   * If dispute is won, the transaction will go back to SETTLED status. We recommend {@link [Subscribing to our webhooks](https://docs.synapsefi.com/api-references/subscriptions)} to be notified.
+   * @param transId Unique ID for transaction 
+   * @param bodyParams Array of supporting docs converted into base 64 encoded strings
+   */
+  disputeChargeback(transId: string, bodyParams: IDisputeChargebackPayload) {
+    const { host, headers } = this;
+    const url = `${host}/trans/${transId}/dispute-chargeback`;
+
+    return axios.patch(url, bodyParams, { headers });
+  }
+
+  /**
+   * @description {@link [Fetches allowed node types](https://docs.synapsefi.com/api-references/nodes/allowed-node-types)}
+   * @returns list of allowed node types
+   */
+  getNodeTypes() {
+    const { host } = this;
+    return axios.patch(`${host}/nodes/types`);
+  }
+
+  /**
+   * @description {@link [Fetches allowed user document types](https://docs.synapsefi.com/api-references/users/allowed-document-types)}
+   * @returns list of allowed user document types
+   */
+  getUserDocumentTypes() {
```

Comment:
is `.patch` expected in these client calls? The document writes that it's a `GET` call

eg:
https://docs.synapsefi.com/api-references/nodes/allowed-node-types

Reply:
good catch - fixed